### DISCUSSION TITLE
Popups being super annoying

### DISCUSSION THREAD

**Discussion by rredford** [community/community#44630] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Every time I mouseover a nick, it will pop up info, so does some other stuff that pops up info box.

That is fine, but there is NO way to get rid of it. If it covers something, I have to find something else to pop info so that spot is uncovered.

What should happen, with info box:
I click anywhere else and it goes away.

What actually happen:
It doesn't go away, blocking part of page.

If click anywhere cant work, add a button to close infobox.

![image](https://user-images.githubusercontent.com/151014/213060651-f6d5e659-04a8-440d-9a4a-5406b9591ab1.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rredford ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by rredford** [reply]
Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - I have passed this on to the proper team - they are aware that this is affecting users in the Firefox browser and working on a fix. When I have updates I'll share them with you.

**Reply by rredford** [reply]
Hm NOW it works fine. Hope it means they fixed it already, and not random fix. Latter suck as flaw is still there lol.

Thanks, hope they find and fix it, if not already.

**Reply by 2bndy5** [reply]
Still seems to be happening but only if the page isn't manually refreshed. My steps to reproduce are:

1. Open Firefox and go to any github.com page, then click a link that targets another page in the github.com domain. Browsing a repo's issues list and viewing a individual issue from that list seems to be the easiest way to do this.
2. Hover mouse over anything that triggers a tooltip (user mentions, hyperlinked PRs/issues, etc) and move the mouse away. Notice the tooltip remains open until another hover area triggers a tooltip.
3. Hit F5 (to refresh page), and notice the tooltips close when the mouse leaves the hover area.

**Reply by rredford** [reply]
Confirmed here also. Steps work to trigger this bug.

**Reply by CapsAdmin** [reply]
Can also confirm. I've had this issue now for what feels like little over a week.

**Reply by icholy** [reply]
I've been toggling the profile drop-down at the top right of the page to close the stuck popups.

**Reply by 2bndy5** [reply]
@icholy Thanks for the workaround tip! This problem has been super annoying!

**Reply by jfbu** [reply]
I have experienced it too on Firefox both on mac os and on Ubuntu 20.04.  If i 1) click on the tooltip then 2) press the Esc key it I can make the tooltip go away but I have to be extra careful to not hit again the same trigger with the mouse...

**Reply by martindholmes** [reply]
I'm also finding this infuriating. How did this get past QA and into the live site? It's INSTANTLY obvious and annoying.

**Reply by jfbu** [reply]
It appears the problem has been solved. EDIT: actually I am still experiencing it although at times it seems to have gone away.  Got the not-to-be-trusted feeling the tooltips on links to PRs were more sticky than tooltips on user info.

**Reply by icholy** [reply]
Still experiencing it.

**Reply by rredford** [reply]
Clicking a link then going back in history also triggers this bug. It's still problemic

**Reply by mjf** [reply]
Still present. Firefox 109.0 (64-bit) on Fedora Linux 36 (Thirty Six). Extremely annoying! :-1:

**Reply by msl3v** [reply]
Can confirm. It has become difficult to interact with github.com

**Reply by jfbu** [reply]
I have now confirmed by repeated trials the following:
- let's hover on some user name or link to another PR, a tooltip appears
- I click on it, in some safe white area, once.
- Then I click somewhere else **on the page** on some safe white area, also once.

The tooltip then goes away, **and** the problem *on that page* (say a page for a PR on a project) appears to be fixed.

**Reply by 2bndy5** [reply]
> I'm also finding this infuriating. How did this get past QA and into the live site? It's INSTANTLY obvious and annoying.

I'm starting to think this problem is a bug in a JS framework that github uses. In which case, they'd likely be waiting on a fix from the framework (upstream). Otherwise, I think the fix would've been applied by now.

@queenofcorgis _Any_ update would be appreciated at this point.

**Reply by queenofcorgis** [reply]
Hi there - they are still working on this, they've been able to reproduce it, but I have nothing further. Appreciate the patience!

**Reply by EBoisseauSierra** [reply]
FYI, the issue appears now fixed for me, running Firefox DevEd 110.0b6 (64-bit).
Thanks to whoever did this :heart:

**Reply by mradcliffe** [reply]
Unfortunately I was still able to reproduce the issue in Firefox 102 ESR (64-bit) using the following the steps provided by 2bnd5 earlier in this thread regarding clicking a link versus refreshing the page.

Edit: Here are complete steps to reproduce I used to verify

1. Go to a repository such as https://github.com/microsoft/vscode-docs which was in my Explore repositories list 
2. Click the "Pull requests" navigation link
3. Click a Pull Request link
4. Move the cursor over the author
5. Move the cursor over the pull request description
- The popover does not go away
6. Refresh the page
7. Move the cursor over the author
8. Move the cursor over the pull request description
- The popover goes away
9. Click the "Pull requests" navigation link
10. Click the Pull Request link again
11. Move the cursor over the author
12. Move the cursor over the pull request description
- The popover does not go away.

**Reply by icholy** [reply]
Let's just assume it's still broken until we get an official update. We know the team is working on it, so these "me too"s and attempts at debugging are just noise in everyone's notifications.

**Reply by jamieshark** [reply]
Hi there! I'm an engineer on the team that is working to resolve this issue. Apologies for the disruption; this behavior was a result of some accessibility changes we are trying to make across the site. We have a potential fix ready and are working towards resolving the issue. Thanks for your patience.

**Reply by jamieshark** [reply]
For those following along at home, my suspicions are that the "bug" has to do with the way that Chrome and Firefox handles retaining focus on elements vs passing it back to the window. 🙂

**Reply by fantasai** [reply]
@jamieshark Could popups be disabled entirely until the fix is properly tested and deployed? At least then the site would be usable. (It's not like :hover tooltips provide any kind of critical functionality, but being able to read and click on things is kind of critical.) Anyway thanks for answering here.

**Reply by jamieshark** [reply]
@fantasai - I think the fix should have been deployed by now (sorry for not answering earlier). Though the user feedback has prompted the team to think about adding this as a user setting, thanks all.

**Reply by rredford** [reply]
Seemed to work, but wasn't certain. :)

**Reply by Yaron10** [reply]
The issue seems to have been fixed. 👍 
Thank you.

**Comment by Yaron10** [comment]
Extremely annoying. Please fix it ASAP.

**Comment by GillesDuvert** [comment]
would greatly appreciate a quick bug fix, as this is terribly annoying.

**Comment by begin-again** [comment]
As a work around on pull requests I have been using the GH extension for VS Code.

**Comment by groner** [comment]
I have a work around.

Clicking inside the tooltip and then clicking again outside gets them to behave for a while (probably until navigation).

**Reply by jfbu** [reply]
👍 but already mentioned at https://github.com/orgs/community/discussions/44630#discussioncomment-4780660

😄  (edit it appears the above link may not work, for this the comments on the most commented reply need to be all displayed it seems)

**Reply by groner** [reply]
Oh, right. I wish I'd seen that earlier in the week.

**Reply by jamieshark** [reply]
This comment was helpful to find the root of the issue, thank you!

**Comment by akosma** [comment]
Is there a setting somewhere to disable them altogether? Not only are they annoying as hell lately, but I'd rather _not_ see them again, at all.

**Reply by 2bndy5** [reply]
Sounds like a good feature request! You should start a new discussion to focus on offering an option to disable the pop-ups...

**Comment by arobotd** [comment]
Yes this is still an issue. One that is driving me more and more insane. A swift fix would be greatly appreciated!

**Comment by hantsy** [comment]
I hope there is a global option in the personal preference to disable it at all, it is noisy.

**Reply by 2bndy5** [reply]
They used the term "fix", not "enhance". Being the second person on this thread to suggest a 'disable pop-ups' option, I highly encourage you open a new discussion to request that feature.

**Reply by CapsAdmin** [reply]
I think what's annoying is just that it covers text sometimes, so when it gets stuck it's extremely annoying. Adding an option to disable it feels like a wrong solution.

You could just block it using a content blocker.

**Reply by crystal-williams-brown** [reply]
> I think what's annoying is just that it covers text sometimes, so when it gets stuck it's extremely annoying. Adding an option to disable it feels like a wrong solution.
> 
> You could just block it using a content blocker.

How would you block it?
